.eslintrc.js

```diff
@@ -2,6 +2,7 @@ var prettierConf = require('./prettier.config.js');
 
 module.exports = {
   extends: ['airbnb', 'prettier'],
+  parser: 'babel-eslint',
   rules: {
```

Comment:
@jourdain sanity check here

Reply:
[This](https://www.npmjs.com/package/@babel/eslint-parser) might be more appropriate. 
What do you think @floryst?